mantidproject/mantid
# Add workflow diagrams for ISIS Powder + Minor refactor

Description of work.
This PR adds several workflow diagrams for ISIS Powder
It also refactors some code + fixes some bugs I noticed:

- Fixed diffing `.dot` files on Windows - Previously it would attempt to open them as Word documents
- Off by one on call to Mask Detectors
- Rearrange order of functions in Pearl Saving routines (no code changes)
- Removed an obsolete level of indirection when getting the monitor spectrum of a workspace 
- Changed a function in common to extract a single workspace to avoid breaking SRP
- Capitalised `Calibration` filename so it correctly orders itself on the documentation page and matches existing conventions. 

**To test:**
Build Docs

Look at: 
*Techniques/ISIS-Powder-Workflows* ensure all diagrams are rendered and look appropriate
*Techniques/ISIS-Powder-Index* ensure a new link appears to previously mentioned page. 

Fixes #19966 

**Release Notes** 

Added to release notes [here](https://github.com/mantidproject/mantid/pull/19983/files#diff-f20ef25841b2c33b446d2e9ba7341f29)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
